src/ophyd_async/epics/pvi/pvi.py

```diff
@@ -296,3 +306,30 @@ async def fill_pvi_entries(
     # We call set name now the parent field has been set in all of the
     # introspect-initialized devices. This will recursively set the names.
     device.set_name(device.name)
+
+
+def pre_initialize_blocks(
```

Comment:
blocks is a PandA specific term, please could we change this to something like:
```suggestion
def create_children_from_annotations(
```